tests/quotes.pyi

```diff
@@ -0,0 +1,28 @@
+from typing import TypeVar, Literal, Annotated
+
+__all__ = ["f", "g"]
+
+
+def f(x: "int"):  # Y020 Quoted annotations should never be used in stubs
+    ...
+
+def g(x: list["int"]):  # Y020 Quoted annotations should never be used in stubs
+    ...
+
+
+_T = TypeVar("_T", bound="int")  # Y020 Quoted annotations should never be used in stubs
+
+def h(x: Literal["a", "b"], y: Literal["c"], z: _T) -> _T:
+    ...
+
+def i(x: Annotated[int, "lots", "of", "strings"]) -> None:
+    """Documented and guaranteed useful."""  # Y020 Quoted annotations should never be used in stubs  # Y010 Function body must contain only "..."
```

Comment:
The error message is a bit confusing: to a human, this is a docstring, not a "Quoted annotation". Maybe we should use two error codes?

Reply:
Added Y021 for docstrings